samples/cpp/benchmark_app/main.cpp

```diff
@@ -578,6 +578,9 @@ int main(int argc, char* argv[]) {
                 set_infer_precision();
             } else if (device.find("GNA") != std::string::npos) {
                 set_infer_precision();
+            } else if (device.find("NVIDIA") != std::string::npos) {
+                setThroughputStreams();
+                set_infer_precision();
```

Comment:
is it possible to unify this place not to have device-specific if else?
I suppose we can set inference precision and streams based on supported_properties.

Reply:
> is it possible to unify this place not to have device-specific if else? I suppose we can set inference precision and streams based on supported_properties.

We have a separate ticket (80788) for that which I created one year ago. I would like to avoid mixing refactoring with NVIDIA support right now.